meta-ros-common/conf/ros-distro/ros-distro.conf

```
@@ -47,7 +47,7 @@ ROS_EXTRA_BUILDCFG_VARS ?= "DISTRO_NAME ROS_DISTRO ROS_VERSION ROS_PYTHON_VERSIO
 BUILDCFG_VARS:append = " ${ROS_EXTRA_BUILDCFG_VARS}"
 
 # Ignore metadata that requires meta-raspberrypi if it's not present.
-BBMASK +=  "${@bb.utils.contains('BBFILE_COLLECTIONS', 'raspberrypi', '', 'recipes-bsp/bootfiles/rpi-config_%.bbappend', d)}"
+BBMASK:append =  "${@bb.utils.contains('BBFILE_COLLECTIONS', 'raspberrypi', '', 'recipes-bsp/bootfiles/rpi-config_%.bbappend', d)}"
```

```suggestion
BBMASK:append =  " ${@bb.utils.contains('BBFILE_COLLECTIONS', 'raspberrypi', '', 'recipes-bsp/bootfiles/rpi-config_%.bbappend', d)}"
```

Leading space is needed when using the `:append` syntax. I have also included this change in my build, can confirm it works.